### DISCUSSION TITLE
Differentiate types of misses for osu!standard scoring

### DISCUSSION THREAD

**Discussion by abraker95** [ppy/osu#15402] [Ideas]
So misses can happen in one of 3 ways:
1. Mistaps - You tap the note within its boundary, but before or after the 50 timing window
2. Misaims - You tap the note within its timing window but outside and close to its boundary (say within 1.5x circle size)
3. Misreads - Misses that don't satisfy the above two conditions

I'd like to have scoring provide counts for each type of miss so they can be factored into pp calc.

<details><summary>on misreads</summary>
Because I know people are going to mention it, yea you can misread patterns resulting in mistap or misaim. I'd like to think of the misread count as pure misreads tho. Those would happen mostly on low AR maps where it's easy to screw up note order. You would end up pressing the wrong note, which would be at wrong timing and nowhere close to the correct note's boundary.
</details>

**Comment by smoogipoo** [comment]
It's possible, but we can't use it for pp calc as we don't have full information on the scores already submitted.
